src/debugging/DockerServerReadyAction.ts

```diff
@@ -24,37 +25,12 @@ const URI_FORMAT = 'http://localhost:%s';
 /* eslint-disable-next-line no-template-curly-in-string */
 const WEB_ROOT = '${workspaceFolder}';
 
-export class ServerReadyDetector extends vscode.Disposable {
-    private static detectors: Map<vscode.DebugSession, ServerReadyDetector> = new Map<vscode.DebugSession, ServerReadyDetector>();
-
+export class ServerReadyDetector extends vscode.Disposable implements DockerServerReadyManager {
```

Comment:
I don't think this type needs to be disposable; it doesn't seem to store any private disposable fields.

Reply:
Yep, turns out the disposable[] is always empty so removed all of that.